libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -133,6 +133,8 @@ export interface IAttributeFilterButtonOwnProps {
     }>;
     // @deprecated
     identifier?: string;
+    // (undocumented)
+    locale?: string;
```

Comment:
Please add some documentation for this props (one or two sentences is enough), so the user knows what the locale is used for there.